HappyAnding/HappyAnding/Views/Components/MyShortcutCardView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MyShortcutCardView.swift
+//  HappyAnding
+//
+//  Created by KiWoong Hong on 2022/10/22.
+//
+
+import SwiftUI
+
+struct MyShortcutCardView: View {
+    
+    let myShortcutIcon: String
+    let myShortcutName: String
+    let mySHortcutColor: String
+    
+    var body: some View {
+        VStack(alignment: .trailing) {
+            Spacer()
+            HStack {
+                Image(systemName: myShortcutIcon)
+                    .frame(alignment: .leading)
+                    .font(.title2)
+                    .foregroundColor(Color.Gray1)
+                    Spacer()
+            }.padding(.bottom, 3)
+            HStack {
+                Text(myShortcutName)
+                    .Subtitle()
+                    .frame(alignment: .leading)
+                    .foregroundColor(Color.Gray1)
+                Spacer()
+            }
+        }
```

Comment:
```suggestion
        VStack(alignment: .leading, spacing: 3) {
            Spacer()
            Image(systemName: myShortcutIcon)
                .font(.title2)
                .foregroundColor(Color.Gray1)
                .padding(.bottom, 3)
            
            Text(myShortcutName)
                .Subtitle()
                .foregroundColor(Color.Gray1)
        }
```

HStack으로 감싸지 않고, VStack에 alignmet와 padding을 줘서 해결할 수 있을 것 같아요!

Reply:
처음에 alignment와 padding으로 했었는데, 이미지가 ShortcutName이 길어지면 오른쪽 끝부분으로 이동하는 버그(?)가 발생해서 HStack으로 감싸주었습니다.